src/families/location_scale.jl

```diff
@@ -113,16 +113,21 @@ function Distributions._rand!(
     return x .+= location
 end
 
-Distributions.mean(q::MvLocationScale) = q.location
+function Distributions.mean(q::MvLocationScale)
+    @unpack location, scale = q
+    return location + scale * Fill(mean(q.dist), length(location))
```

Comment:
What does this do in a case where e.g. `location` is of dimension 3 and `q.dist` is of dimension 2? Or worse, have those two numbers be the other way around? Or does something (the constructor?) already earlier catch that?

Reply:
Should be fixed by forcing `q.dist` to always be univariate.